core-bundle/contao/templates/twig/page/regular.html.twig

```diff
@@ -8,6 +8,7 @@
 
 {% set body_attributes = attrs()
     .set('id', 'top')
+    .addClass(contao.page.cssClass)
```

Comment:
```suggestion
    .addClass(contao.page.cssClass|default)
```

Just in case that field somehow does not exist?

Reply:
Not really needed, as the `__get` method of the `PageModel` will just return `null`.